test/controller_test.go

```diff
@@ -139,8 +139,8 @@ func TestGetTestResourcesFromClients(t *testing.T) {
 				t.Fatal(err)
 			}
 			if !reflect.DeepEqual(*actualTestResources, tt.testResources) {
-				t.Errorf("GetTestResourcesFromClients = %v, expected %v", *actualTestResources, tt.testResources)
-				t.Log(cmp.Diff(*actualTestResources, tt.testResources))
+				t.Errorf("GetTestResourcesFromClients() Fail")
+				t.Logf("Diff request body: -want +got: %s", cmp.Diff(tt.testResources, *actualTestResources))
```

Comment:
nit: I think we should probably change the `reflect.DeepEqual()` above to be `cmp.Diff()` to simplify this.